submissions/YuliiaHordiichuk/js-dom/script.js

```diff
@@ -0,0 +1,102 @@
+const data = [
+    {
+        title: 'Exotic',
+        desc: 'Exotics are essentially the shorthaired version of the Persian cat. Like their longhaired cousins, these cats are calm and affectionate, preferring a warm lap or plush cushion to intense activity. They have a moderate activity level and will happily get out some energy with cat toys before heading back to your lap.',
+        img: './img/exotic.jpg',
+        subtitle: 'Breed Overview',
+        deatils: ['Height: 10 to 12 inches', 'Weigth: 10 to 12 pounds', 'Physical Characteristics: short legs, rounded body, short nose']
```

Comment:
It's a typo here. Use `deatils` here, please

Reply:
done